packages/landing/src/index.ts

```diff
@@ -20,4 +21,10 @@ document.addEventListener('DOMContentLoaded', async () => {
   window.basemapsUi = ui;
 
   if (basemaps.config.debug) addDebugLayer(basemaps);
+
+  SplitIo.getClient().then((client) => {
+    if (client == null) return;
+
+    // const isLayerSwitcherOn = SplitIo.getTreatment(SplitTreatment.LayerSwitcherButton);
```

Comment:
Is this comment still useful?

Reply:
Was more to show how it's used, I'm working on the next pr which does something with it